doc/docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -61,27 +63,27 @@ neo4j.authentication.basic.username neo4j
 
 
 
-== Neo4j Driver Options
+== Neo4j driver options
 
-Under the covers, the spark connector uses the link:https://neo4j.com/docs/driver-manual/current/get-started/#driver-get-started-about[official Neo4j Java Driver].  As such, in many situations you'll want the control to set driver options to account for your production deployment of Neo4j, and how to communicate with it.   This is done using the `options` example above.
+Under the covers, the Spark connector uses the link:https://neo4j.com/docs/java-manual/current/get-started/#java-driver-get-started-about[official Neo4j Java Driver].  In many situations you want the control to set driver options to account for your production deployment of Neo4j, and how to communicate with it. This can be done using the `options` example above.
 
 The following table captures the most common configuration settings to use with the Neo4j driver.  For full
-documentation on all possible configuration options for Neo4j drivers, see the link:https://neo4j.com/docs/driver-manual/current/client-applications/#driver-configuration[Neo4j Drivers Manual].
+documentation on all possible configuration options for Neo4j Drivers, see the link:https://neo4j.com/docs/drivers-apis/#_neo4j_drivers[Neo4j Drivers Manuals].
```

Comment:
I have changed the link, because the previous one pointed to the Docs landing page. Or might it point to any specific chapter of the Java Driver manual? The design of the drivers docs has been changed, and now we have separate manuals for the supported drivers

Reply:
@NataliaIvakina Please just point the Java driver docs! Thank you!